File: python-prov/prov_value.py
import datetime

import prov.dot
from prov.model import ProvDocument
import prov.graph

def perform():
    document = ProvDocument()
    ns = document.add_namespace("ex", "https://example.org/")
    a = document.agent("ex:a",{"prov:value":4})
    document.serialize(r"..\java-prov\prov_value.provn", format="provn")
    print(document.get_provn())

File: java-prov/src/main/java/org/example/ProvValue.java
package org.example;

import org.openprovenance.prov.interop.InteropFramework;
import org.openprovenance.prov.model.Document;


public class ProvValue implements TestCase {

    public void serialize(String format) {
        //TODO
    }

    public void deserialize(String format) {
        var inf = new InteropFramework();
        var document = inf.readDocumentFromFile(String.format("data/prov_value.%s", format));
    }
}
